File: CursoPOO/Java/Car.java
class Car {
    //Atributos
    private Integer id;
    private String license;
    private Driver driver;
    protected Integer passengers;
    //Metodo constructor
    public Car(String license, Driver drive){
        this.license = license;
        this.driver = drive;
        

    }
    //Metodo
    void printDataCar() {
        if(passengers != null){
            System.out.println("License: " + license + "\n" + "Name Driver: " + driver.name + "\n" + "Passengers: " + passengers);
        }
        
    }


    public Integer getPassegengers(){
        return passengers;
    }

    public void setPassengers(Integer passengers) {
        if(passengers == 4){
            this.passengers = passengers;
        } else {
            System.out.println("Necesitas asignar 4 pasajeros");
        }
    }
    public Integer getId() {
        return id;
    }
    public void setId(Integer id) {
        this.id = id;
    }
    public String getLicense() {
        return license;
    }
    public void setLicense(String license) {
        this.license = license;
    }
    public Driver getDriver() {
        return driver;
    }
    public void setDriver(Driver driver) {
        this.driver = driver;
    }
    
    
    
}

File: CursoPOO/JS/car.js


export default class Car {
    constructor(license,driver){
        this.id;
        this.license= license;
        this.driver = driver;
        this.__passengers
    }


    get passengers(){
        return this._passengers
    }

    set passengers(passengers){
        if(passengers != 4) {
            console.log("Debes tener 4 pasajeros")
        } else {
            this._passengers = passengers;
        }
    }


    //Metodo para imprimir 

    printDataCar = () => {
        console.group(this.id)
        console.log('License: ', this.driver)
        console.log(this.driver)
        //console.log(this.driver.document)
        console.log('Passengers: ' + this.passengers )
        console.groupEnd()
    }
}